src/care_communication/api/auth/kiosk_auth.py

```
@@ -21,11 +27,11 @@ def authenticate(self, request):
             raise AuthenticationFailed("User is not authorized to access patient data")
 
         try:
-            cleaned_dob = datetime.strptime(dob, "%Y-%m-%d").date()
+            birth_year = int(birth_year)
         except ValueError:
-            raise AuthenticationFailed("Invalid date format. Expected YYYY-MM-DD")
+            raise AuthenticationFailed("Invalid birth year")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)